Services/DataX.Flow/Flow.ManagementService/ServiceCollectionExtensions.cs

```diff
@@ -23,9 +25,9 @@ public static class ServiceCollectionExtensions
         /// <param name="exportTypes">Types being exported</param>
         /// <param name="instanceExports">Instances that are created outside that need to be added to the container</param>
         /// <returns></returns>
-        public static IServiceCollection AddMefExportsFromAssemblies(this IServiceCollection services, ServiceLifetime lifetime, IEnumerable<Assembly> assemblies, Type[] exportTypes, object[] instanceExports)
+        public static IServiceCollection AddMefExportsFromAssemblies(this IServiceCollection services, ServiceLifetime lifetime, IEnumerable<Assembly> assemblies, Type[] exportTypes, object[] instanceExports, ILoggerFactory loggerFactory = null, bool local = false)
         {          
-            var configuration = new ContainerConfiguration().WithAssemblies(assemblies).WithProvider(new InstanceExportDescriptorProvider(instanceExports));
+            var configuration = new ContainerConfiguration().WithAssemblies(assemblies).WithProvider(new LoggerExportDescriptorProvider(instanceExports, loggerFactory));
```

Comment:
Should probably rename this class. Name gives the impression that this handles only logger instances which is what I assumed initially, but then after looking at the implementation saw that this handles non-logger instances as-well.